guides/doc-Managing_Hosts/topics/proc_registering-a-host-to-project-using-the-global-registration-template.adoc

```diff
@@ -0,0 +1,174 @@
+* You must have root privileges on the host that you want to register.
+
+.Procedure
+. In {Project} web UI, navigate to *Hosts* > *Provisioning Templates* and locate and click *Linux registration default*.
+
+. Click the *Association* tab.
+
+. Select the operating systems that you want to register.
+
+. Navigate to *Hosts* > *Operating Systems* and click the operating system that you want to register.
+
+. Click *Submit*.
+
+. Click the *Templates* tab.
+
+. From the *Registration template* list, select *Linux registration default*.
+
+. Click *Submit*.
+
+. Navigate to *Hosts* > *All Hosts* > *Register Host*.
+
+. Optional: If you want to register hosts to {SmartProxy}, enter the following command on the {SmartProxy} to enable required modules:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# foreman-installer --foreman-proxy-registration \
+--foreman-proxy-templates \
+--foreman-proxy-template-url 'http://{smartproxy-example-com}'
+----
+
+. What fields are required for Satellite and Foreman?
+
+. Optional: From the *Host Group* list, select the host group to associate the hosts with.
+
+. From the *Operating System* list, select the operating system of hosts that you want to register.
+
+. Optional: If you want to register hosts to a {SmartProxy}, from the *{SmartProxy}* list, select the {SmartProxy} to register hosts to.
+
+ifeval::["{build}" == "satellite"]
+. Optional: If you do not want to register hosts with Insights, from the *Insights* list, select *No*. What are the defaults for those two?
+endif::[]
+
+. Optional: If you do not want to deploy {Project} SSH keys to hosts, from the *Remote Execution* list, select *No*.
+
+. Optional: {Project} uses the JSON Web Tokens (JWT) for authentication.
+This duration of this token defines how long the generated `curl` command works.
+If you want to change the duration of the token, enter the required duration to the *Token lifetime (hours)* field.
+You can set the duration to any number of hours.
++
+Note that {Project} uses the permissions of the user who generated the `curl` command for authorization of hosts.
+If user loses or gains additional permissions, the permissions of the JWT change too.
+Therefore, do not delete, block, or change permissions of the user during the token duration.
+
+. Optional: What is remote execution interface?
+
+ifeval::["{build}" != "satellite"]
+. In the *Activation Key(s)* field, enter one or more activation keys to assign to registered hosts.
+endif::[]
+
+. Click *Generate command*.
+
+. Copy the generated `curl` command to enter it on the hosts.
+
+. On the hosts that you want to register, enter the generated `curl` command as `root`.
+
+.SSL
+For calling Foreman endpoints via https CA authority must be installed on the host.
+Path to CA file can be found in Settings > Authentication > SSL CA file.
+
+If you don't want to upload & install CA on the host, you can run first `curl` command
+with `---insecure` parameter, like this:
```

Comment:
@stejskalleos is it safe to tell Satellite users to run `curl` with the `insecure` option? Although we tell users to do this in the official Satellite docs [1] now, is there a better and safer way of doing this?

[1] See step 2 in https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html-single/managing_hosts/index#registering-a-host-to-satellite

Reply:
Well, there is always a risk that something could happen, but I would say it's okay.

If they don't like the idea of insecure call, they can manually install certification authority (File defined in _Settings > Authentication > SSL CA file_) on the registered machine and then they can securely call Satellite instance without `--insecure`

Reply:
How to install this CA file on hosts? Is that possible to describe this shortly or it is better to leave this decision on "how" to each user?

Reply:
It's pretty easy:

* Download file from _Settings > Authentication > SSL CA file_
* Copy the file to the `/etc/pki/ca-trust/source/anchors/`
* Run command `update-ca-trust extract`